examples/demo_mock.ipynb

```diff
@@ -0,0 +1,140 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "wkR0YvENQni4"
+      },
+      "source": [
+        "**Setup dependencies**"
```

Comment:
Perhaps we can move this file into `SMARTS/examples/env` folder and rename it to `SMARTS/examples/env/create_run_visualize.ipynb`.